microstrain_inertial_driver/CMakeLists.txt

```diff
@@ -181,4 +181,5 @@ install(FILES ${BOOST_LIBS} ${MSCL_DIR}/libmscl.so
 #############
 
 ## Run roslint on only the node files for now
+set(ROSLINT_CPP_OPTS "--filter=-whitespace/line_length,-runtime/references,-readability/fn_size")
 roslint_cpp(${LIB_SRC_FILES} ${LIB_INC_FILES} ${NODE_SRC_FILES} ${NODE_INC_FILES})
```

Comment:
This looks good, but I think a good thing to do here to hopefully catch these errors before they get merged would be to run roslint on the buildfarm, which could be done by adding the following code snippet around this line:
```cmake
if(BUILD_TESTING)
  set_property(TARGET roslint PROPERTY EXCLUDE_FROM_ALL OFF)
endif()
```

And then somewhere near the top of the `CMakeLists.txt`, add the line
```cmake
option(BUILD_TESTING "Whether or not to build tests" OFF)
```

Reply:
Ok following up with https://github.com/LORD-MicroStrain/microstrain_inertial/pull/154